steps/lambdatest-upload/1.0.1/step.yml

```diff
@@ -0,0 +1,83 @@
+title: LambdaTest Upload
+summary: |
+  Upload APK/IPA files to LambdaTest's App Automate
+description: |
+  Upload your build files to LambdaTest for remote device testing on Real Devices. Upload your IPA/APK file to LambdaTest App Automation and get back the app's url for your remote device tests.
+website: https://github.com/LambdaTest/bitrise-step-lambdatest-upload
+source_code_url: https://github.com/LambdaTest/bitrise-step-lambdatest-upload
+support_url: https://github.com/LambdaTest/bitrise-step-lambdatest-upload/issues
+published_at: 2022-04-21T23:40:48.586584+05:30
+source:
+  git: https://github.com/LambdaTest/bitrise-step-lambdatest-upload.git
+  commit: 0566aed52e765f808970fd472b4af8af1b26925a
+project_type_tags:
+- ios
+- macos
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  - name: jq
+  - name: ack
+  apt_get:
+  - name: curl
+  - name: jq
+  - name: ack
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+    is_required: true
+    summary: The app file you want to upload to LambdaTest, usually `$BITRISE\_APK\_PATH`
+      or `$BITRISE\_IPA\_PATH`
+    title: A Bitrise generated APK or IPA path
+  upload_path: null
+- lambdatest_username: null
+  opts:
+    description: |
+      The username you use to log into your LambdaTest account
+    is_required: true
+    is_sensitive: true
+    summary: Your LambdaTest username
+    title: LambdaTest username
+- lambdatest_access_key: null
+  opts:
+    description: |
+      The access key you use to log into your LambdaTest account
+    is_required: true
+    is_sensitive: true
+    summary: Your LambdaTest access key
+    title: LambdaTest access key
+- app_name: null
+  opts:
+    description: |
+      The name of your application
+    is_required: true
```

Comment:
Adding a new `required` input with no default value is considered a "breaking change" so according to semver the version of the step will have to be `2.0.0`